docs/STUDY_PUBLISHING.md

```diff
@@ -20,16 +10,11 @@ The Rally Core extension checks the Firefox RemoteSettings server for a list of
 For local testing, see: https://github.com/mozilla-rally/core-addon#building-and-testing-a-study-locally
 There is also a RemoteSettings dev server you may use for local testing / QA: https://remote-settings.readthedocs.io/en/latest/tutorial-dev-server.html
 
-Updates to the staging and production RemoteSettings servers must receive sign off. The current reviewers are @knowtheory and @rhelmer.
-
-## Updating the Study Extension
-
-First, refer to the [Build your extension](#build-your-extension) section. Make sure to update your version numbers!
-
-Once you have your extension ready to publish, the usual addons.mozilla.org update process applies - log in to https://addons.mozilla.org and navigate to the "Developer Hub", then follow the instructions to update your extension.
+Updates to the staging and production RemoteSettings servers must receive sign off. The current reviewers are @knowtheory, @rhelmer, and @Dexterp37.
 
-NOTE - you may publish as a "pre-release" which will prevent existing users from updating until QA is complete.
+You can find an example of the expected JSON output of the RemoteSettings in the [locally-available-studies.json](https://github.com/mozilla-rally/rally-core-addon/blob/master/public/locally-available-studies.json) file of the core add-on repository. Note that this file must conform to the [studies-schema.json](https://github.com/mozilla-rally/rally-core-addon/blob/master/public/studies-schema.json) in the same repository.
 
-It may take some time for your updated extension to be reviewed and available for download.
+The JSON may be edited directly, but since the RemoteSettings server is aware of the above schema, a simple HTML form may be used instead.
```

Comment:
Is there any documentation on how to access that/how to change it?

Reply:
Yeah, it's the default, maybe I should make that clear?

Reply:
If you could add a link here that would be helpful for future onboarding of other people.